identity/app/views/completeConsents.scala.html

```diff
@@ -49,15 +47,6 @@ <h1 class="identity-upsell-title">Before you go...</h1>
             </form>
         }
 
-        @request.cookies.get("SC_GU_GUEST_PW_SET").map { passwordSetToken =>
-            <div
-                class="js-identity-upsell-account-creation"
```

Comment:
If you search the project for `js-identity-upsell-account-creation` I think there are more things to delete - (the JavaScript that renders the password input)

Reply:
good spot!